contexts/DelegationContext.tsx

```diff
@@ -234,40 +236,108 @@ export function DelegationProvider({ children }: { children: ReactNode }) {
       {
         voting,
         delegateAddress,
+        notificationMessage: `Requesting ${truncateEthAddress(delegateAddress)} to be your delegate...`,
```

Comment:
The logic for the delegation mutations is slightly more complicated because we need more information about what's happening to create the message. The same mutations are used in different contexts, so we must specify the context when calling the function.